source/Camera/Basic/CaptureAssistant/CaptureAssistant.cpp

```diff
@@ -14,7 +14,7 @@ int main()
         std::cout << "Connecting to camera" << std::endl;
         auto camera{ zivid.connectCamera() };
 
-        const auto resultFile{ "result.zdf" };
+        const auto *resultFile{ "result.zdf" };
```

Comment:
Inconsistent style, avoid this style.

```suggestion
        const auto *resultFile = "result.zdf";
```

Reply:
This is a copied sample. I'll report in zivid-sdk. Will still take the change.

Reply:
Best if you make a small PR on zivid-sdk for this change as well.